samples/aws/cluster-template.yaml

```diff
@@ -758,7 +853,7 @@ data:
                       key: access_key
                       name: aws-secret
                       optional: true
-              image: registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.2.0
+              image: registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.25.0
```

Comment:
this is way too new compared to what we had previously, is it an intended change?

Reply:
CSI has a hard dependency on external CCM configuration, old ones will not work or will have unintended side-effects